src/scripts/uploadPremadeBundles.js

```diff
@@ -63,12 +116,16 @@ const getBundleFiles = (directory, searchPattern) => {
 async function main() {
   await mongoUtil.client.connect();
   console.log('Connected successfully to server');
+
+  // --test assumes default for all other parameters
+  const test = process.argv[2] === '--test';
```

Comment:
Totally fine if the answer is no, but I don't think this `--test` works if you use the `dirPath` input?

Reply:
Correct- it assumes the default for the directory path.